recipes/optuna-dashboard/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "optuna-dashboard" %}
+{% set version = "0.3.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/optuna-dashboard-{{ version }}.tar.gz
+  sha256: 277a81b0ebb33f405dfba2cd15d2ec628abd7b4acacd6364c88cea04ee3fa907
+
+build:
+  number: 0
+  entry_points:
+    - optuna-dashboard = optuna_dashboard.cli:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - bottle
+    - optuna >=2.4
+    - python
+    - typing-extensions  # [py<38]
+
+test:
+  imports:
+    - optuna_dashboard
+    - tests
+  commands:
+    - pip check
+    - optuna-dashboard --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/optuna/optuna-dashboard
+  summary: Real-time dashboard for Optuna.
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - c-bata
+    - hadim
```

> @c-bata: I have added you as a maintainer, in consequence, you'll have full permission on the feedstock repo. There is no need to close this PR. We only need the cython dep to be added on the optuna package.
> 
> (I can also remove myself as a maintainer if that's an issue for you)

@hadim Sorry for the delayed response. Could you update maintainers like the following?

```suggestion
    - c-bata
    - g-votte
    - HideakiImamura
    - hvy
    - iwiwi
    - not522
    - sile
    - smly
    - toshihikoyanase
    - ytsmiling
```